src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -171,60 +172,68 @@ docker run <OTHER-DOCKER_OPTIONS> -e CHE_HOST=<ip-addr-or-dns> eclipse/che:<vers
 
 *Connectivity*
 
-The browser client initiates communication with the Che server by connecting to `che-ip`. This IP address must be accessible by your browser clients. Internally, Che runs on Tomcat which is bound to port `8080`. This port can be altered by setting `CHE_PORT` during start or in your `che.env`.
+The browser client initiates communication with the Che server by connecting to the `che-ip`. This IP address must be accessible by your browser clients. Internally, Che runs on Tomcat that is bound to the port `8080`. This port can be altered by setting the `CHE_PORT` variable during start or in the `che.env` file.
 
-When a user creates a workspace, the Che server connects to the Docker daemon at `docker-ip` and uses the daemon to launch a new set of containers that will power the workspace. These workspace containers will have a Docker-configured IP address, `workspace-container-ip`. The `workspace-container-ip` isn’t usually reachable by your browser host, `docker-ip` will be used to establish the connections between the browser and workspace containers.
+When a user creates a workspace, the Che server connects to the Docker daemon at `docker-ip` and uses the daemon to launch a new set of containers that power the workspace. These workspace containers have a Docker-configured IP address, `workspace-container-ip`. The `workspace-container-ip` is not usually reachable by your browser host. And, `docker-ip` is used to establish the connections between the browser and workspace containers.
 
-Che server will provide workspace containers with following environment variables:
+Che server provides workspace containers with the following environment variables:
 
-* `CHE_API_INTERNAL`, points to the internal API endpoint, that is accessible across other machines within the workspace. Value is taken from Che server `CHE_INFRA_DOCKER_MASTER__API__ENDPOINT` variable, which can be initialized either by CLI or default value from `che.properties`.
-* `CHE_API_EXTERNAL`, points to the external API endpoint, that is used by browser clients. Value is taken from Che server `CHE_API` variable. It’s default value is defined in `che.properties`.
-* `CHE_API` will point to the same value as `CHE_API_INTERNAL` for backward compatibility (will be removed in the future).
+* `CHE_API_INTERNAL`: Points to the internal API endpoint that is accessible across other machines within the workspace. Its value is taken from the Che server `CHE_INFRA_DOCKER_MASTER__API__ENDPOINT` variable that can be initialized either by the CLI or default value from the `che.properties` variable.
```

Comment:
I'd keep the wording here consistent with the other documented environment variable (the wording is less ambiguous):

"or the default value defined in the `che.properties` variable."